.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,13 +2,13 @@
 name: Bug report
 about: Create a report for a bug or to help us improve
 title: A brief clear title describing the bug from a user's perspective
-labels: bug, needs triage
-assignees: ''
-
+labels: bug, needs triage, needs daily triage
```

Comment:
needs both labels? I would have thought that just `needs daily triage` is enough and
the `needs (weekly) triage` can be added if required - clearly I'm not understanding the intended process here!

are you expecting that all bugs would typically be triaged twice?

Reply:
Would need to filter both in triage sessions... or have both removed by the person triaging the bug 👀 ......

Reply:
> are you expecting that all bugs would typically be triaged twice?

Yep - daily triage is just to capture anything for hotfix (and maybe assign a preliminary severity to help speed up weekly triage?) Otherwise wouldn't expect release team to triage all bugs, they may not have all the context - this should still go through main weekly triage team 🙏 

I will add to release management docs!

Reply:
For reference! https://github.com/msupply-foundation/open-msupply/wiki/Release-process#daily-bug-triage